app/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
           xmlns:android="http://schemas.android.com/apk/res/android"
           xmlns:tools="http://schemas.android.com/tools"
           android:versionCode="106"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/6aa865dbc1d52ef1241c4df238c229c0d05922f0/app/AndroidManifest.xml#L5"></span>This <code>versionCode</code> value (<code>106</code>) is not used; it is always overridden by the value specified in the Gradle build script (<code>1</code>)</td>
    </tr>
  </tbody>
</table>